### PR TITLE
[CDAP-18910] improve file connection markdown

### PR DESCRIPTION
**Pull request by sumengwang**
[CDAP-18910](https://cdap.atlassian.net/browse/CDAP-18910)

image

image

### CODE REVIEW

**Comment by yaojiefeng** [comment]
These are here because when browsing to a file, UI needs this to show the format and encoding, so we shouldn't remove it

**Comment by yaojiefeng** [comment]
It is better to have UI to not show the label if no properties to display, UI relies on the plugin spec to show the properties, so if it sees no properties under the label, better just hide the entire label

**Review by de-lan — APPROVED** [review]

File: core-plugins/docs/File-connector.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
-# File Connector
+# File Connection
```
Comment (de-lan) [inline core-plugins/docs/File-connector.md:1]: Is this change on purpose?

Comment (sumengwang) [reply core-plugins/docs/File-connector.md:1]: yes requested by Robin
